### PR TITLE
Compiler

### PR DESCRIPTION
**Pull request by hihiboss**
How about 'translate' package name?

I agonized over whether to make name to 'compile' or 'translate'.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Contribution 방법
+
+0.  https://github.com/DE-labtory/koa 을 `fork`하기
+1.  작업 할 내용 이슈에 등록하기 or 이슈에 이미 등록된 작업 선택 (작업중인 이슈에 댓글로 표시)
+2.  이슈로 등록된 작업 수행
+3.  수행한 작업의 문서, 단위 테스트 필수로 작성하기
+4.  DE-labtory/koa 폴더에서 `go test -v -p 1 ./... `로 테스트 수행, 
+5.  goimports로 코드 포맷팅 ```goimports -w ./```
```
Comment (zeroFruit) [inline CONTRIBUTING.md:8]: I think `go test -v ./...` is enough now

**Comment by zeroFruit** [comment]
imo `translate` is better

**Comment by junbeomlee** [comment]
Please apply goimports!

File: CONTRIBUTING.md (hunk 1 above)
Comment (junbeomlee) [reply CONTRIBUTING.md:8]: I agree with zeroFruit

**Comment by junbeomlee** [comment]
The structure of the ByteCode and the compiler should be discussed in more detail.

**Comment by AgwaB** [comment]
Why did you separate the ByteCode and the compiler?

File: CONTRIBUTING.md (hunk 1 above)
Comment (hihiboss) [reply CONTRIBUTING.md:8]: Oh, i just copied it from swim... thanks!

**Comment by hihiboss** [comment]
@leesd556 
Let's discuss about that in the meeting!

**Review by junbeomlee — APPROVED** [review]

**Pull request merged**